addons/web/static/src/core/position_hook.js

```diff
@@ -101,6 +103,12 @@ function getBestPosition(reference, popper, { container, margin, position }) {
     const refBox = reference.getBoundingClientRect();
     const contBox = container.getBoundingClientRect();
 
+    // Get the offsets of the containing block (relative to the viewport)
+    // https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block
+    const popStyle = window.getComputedStyle(popper);
```

Comment:
I don't understand why is this needed ?
We call this `getBestPosition` function right after setting the top and left style properties to "0px".
Can you explain ?

Reply:
Yeah it's not needed I made my change while thinking that `getBestPosition` could be exported but if we never plan to export it then we can use the fact that we reset to `0px` before calling it